### PR TITLE
Improvemnt : get quiz questions, teks and style revision

### PR DESCRIPTION
**Pull request by bangmar**
- [x] fix landing hero teks
- [x] fix footer adderess
- [x] improve administration style
- [x] change acara kampus to acara unik
- [x] improve contact form
- [x] get quiz questions from api

### CODE REVIEW

File: apps/lxp/modules/administrations/module.tsx [hunk 1]
```diff
@@ -3,17 +3,17 @@ import { ClientProvider } from "../common/provider";
...
   return (
     <ClientProvider>
-      <div className=" bg-[#F5F5F5] w-full">
-        <BreadCrumb items={administrationBreadCumbs} textColor="text-version2-400" />
-        <div className="bg-[#FFFF] container mx-auto rounded-md p-10 min-h-screen ">
-          <TitleAdministration />
-          <ContentAdministration />
-        </div>
+      <BreadCrumb items={administrationBreadCumbs} textColor="text-version2-400" />
+      <div className="bg-[#FFFF] mx-6 md:mx-14 lg:mx-16 rounded-md p-10 mb-14 min-h-[100vh] ">
```
Comment (maulanasdqn) [inline apps/lxp/modules/administrations/module.tsx]: ini bg-white aja kenapa harus FF?

File: apps/lxp/modules/administrations/section/ContentAdministration/content.tsx [hunk 2]
```diff
@@ -3,9 +3,9 @@ import PrivateInformationAccordion from "../../common/PrivatInformationAccordion
 
 export const ContentAdministration = (): ReactElement => {
   return (
-    <div className="mt-[40px] ">
+    <div className="mt-[32px] ">
       <div className="flex justify-center">
-        <div className={"bg-[#FFFF] rounded-[8px] md:w-[95%] w-full"}>
```
Comment (maulanasdqn) [inline apps/lxp/modules/administrations/section/ContentAdministration/content.tsx:8]: ini juga

File: libs/modules/src/features/features.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ export const FeaturesModule: FC<TFeaturesPageProps> = ({ accessFrom }): ReactEle
   }, [accessFrom, setFeatures]);
 
   return (
-    <section className=" bg-neutral-100/40 min-h-[100vh]">
+    <section className=" bg-[#FAFAFA] min-h-[100vh]">
```
Comment (maulanasdqn) [inline libs/modules/src/features/features.tsx]: jadiin color di tailwind config ya jangan gini

**Review by maulanasdqn — CHANGES_REQUESTED** [review]

File: apps/lxp/modules/administrations/module.tsx (hunk 1 above)
Comment (bangmar) [reply apps/lxp/modules/administrations/module.tsx]: bukan gua :) oke la di benerin

**Review by maulanasdqn — APPROVED** [review]

**Review by maulanasdqn — APPROVED** [review]

**Review by maulanasdqn — APPROVED** [review]

**Pull request merged**